server/cherry_pick.go

```diff
@@ -106,9 +107,20 @@ func (s *Server) doCherryPick(ctx context.Context, version string, milestoneNumb
 	if pr.MergeCommitSHA == "" {
 		return "", errors.Errorf("can't get merge commit SHA for PR: %d", pr.Number)
 	}
+
+	if s.Config.RepoFolder == "" {
+		return "", errors.Errorf("path to folder containing local checkout of repositories is not set in the config")
+	}
+	repoFolder := filepath.Join(s.Config.RepoFolder, pr.RepoName)
+
+	if s.Config.ScriptsFolder == "" {
+		return "", errors.Errorf("path to folder containing the cherry-pick.sh script is not set in the config")
+	}
+	cherryPickScript := filepath.Join(s.Config.ScriptsFolder, "cherry-pick.sh")
```

Comment:
Maybe the empty checks should be moved to the config loader?

Reply:
This is a good candidate for @isacikgoz's config validation library. Low priority, but good to have.